src/jit/utils.cpp

```diff
@@ -2038,6 +2038,68 @@ float FloatingPointUtils::round(float x)
     return *reinterpret_cast<float*>(&bits);
 }
 
+bool FloatingPointUtils::isNormal(double x)
+{
+    int64_t bits = reinterpret_cast<int64_t&>(x);
+    bits &= 0x7FFFFFFFFFFFFFFF;
+    return (bits < 0x7FF0000000000000) && (bits != 0) && ((bits & 0x7FF0000000000000) != 0);
+}
+
+bool FloatingPointUtils::isNormal(float x)
```

Comment:
I do not like this bit checks, but without C++ isnormal I do not see any better solution.